src/ReachSets/reach_blocks.jl

```diff
@@ -339,3 +339,63 @@ function reach_blocks!(ϕ::SparseMatrixExp{NUM},
 
     return nothing
 end
+
+# lazy_X0, dense, with input
+function reach_blocks!(ϕ::AbstractMatrix{NUM},
+                      Xhat0::LazySet{NUM},
+                      U::ConstantNonDeterministicInput,
+                      overapproximate::Function,
+                      n::Int,
+                      N::Int,
+                      blocks::AbstractVector{Int},
+                      partition::AbstractVector{<:AbstractVector{Int}},
+                      res::Vector{CartesianProductArray{NUM}}
+                      )::Void where {NUM}
+    print("using lazy x0 dense with input \n")
```

Comment:
I would avoid this. At least use `info`.

Reply:
i'm still debugging :D